templates/addons/windows/calico/calico.yaml

```diff
@@ -11,7 +11,7 @@ kind: ConfigMap
 apiVersion: v1
 metadata:
   name: calico-static-rules
-  namespace: kube-system
```

Comment:
@marosset this is important to note: because the Calico Helm chart puts everything in `calico-system` namespace, it requires the windows components to be in the same namespace which is why I changed all these manifests. We might want to check if these are being used anywhere outside CAPZ so this doesn't break users / tests when it merges...